### ISSUE TITLE
2018-01-12 Node.js End User Feedback (NEUF) Meeting

### ISSUE THREAD

**Issue by dshaw** [nodejs/user-feedback#26]
# 2018-01-12 Node.js End User Feedback (NEUF) Meeting

## Time

Friday, 2018-01-12 at 17:00 UTC / noon ET / 11am CT / 9am PT

## Links

* **Minutes Doc**: https://docs.google.com/document/d/1igMLOTd0TJHf0hbXqcLIl-tXmj9vlK9G3h1W2d9TQFo/edit
* **Previous Minutes Doc**: https://docs.google.com/document/d/1kc6PTz9o3eAS3ZZKsIMpXD2UUIoeNjYh7dZ4R9ee8Is/edit

## Agenda

Extracted from [neuf-agenda](https://github.com/nodejs/user-feedback/labels/neuf-agenda) labelled issues and pull requests from the [user-feedback](https://github.com/nodejs/user-feedback) repo prior to the meeting.

* Intro / Attendance
* [Key Initiatives](https://github.com/nodejs/user-feedback/labels/key-initiative)
  * 2017 Benchmarking WG Survey (ref: https://github.com/nodejs/user-feedback/issues/22)
  * End User Feedback - Looking for Participants (ref: https://github.com/nodejs/user-feedback/issues/20)
  * Enterprise Advisory <strike>Group</strike> Team (ref: https://github.com/nodejs/user-feedback/issues/18)
* Enterprise Advisory Group Charter, expectations and responsibilities (ref: https://github.com/nodejs/enterprise-advisory-group/pull/5)
* Request for membership by @jchip, WalmartLabs (ref: https://github.com/nodejs/enterprise-advisory-group/issues/7)
* 2018-02-09 Public User Feedback Meeting - Enterprise User Feedback (ref: https://github.com/nodejs/user-feedback/issues/27)

## Invited

* Dan Shaw (@dshaw - CommComm observer)
* Mike Hostetler (@mikehostetler)
* Mihai Ene-Pietrosanu (@mihaiep)
* William Kapke (@williamkapke - CommComm)
* Tierney Cyren (@bnb - CommComm Co-Chair)
* Michael Dawson (@mhdawson)
* Greg Wallace (@gtewallace)
* James Snell (@jasnell)

## Public participation

The Node.js project live streams meetings so anyone can tune in. Everyone is invited to participate in the live chat. The stream will start playing at https://www.youtube.com/c/nodejs+foundation/live once we go live.

There's usually a short cat-herding time at the start of the meeting and occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and we'll be online shortly. ❤️

## Joining the meeting

Join link for invited members and observers: https://t.co/mbDVndWyuZ

There is a standing invitation for all @nodejs/TSC and @nodejs/community-committee members to attend in observer capacity.

**Comment by mhdawson** [comment]
@dshaw hope you don't mind I edited description to add myself to the invitees.

**Comment by dshaw** [comment]
np. Not sure why you weren't already listed.

Built agenda.

**Comment by dshaw** [comment]
@mhdawson or @bnb Can you start Zoom?

**Comment by dshaw** [comment]
PSA: There's an issue with Zoom this morning. We may need to transition to Hangouts. Sorry for the delay.

**Comment by dshaw** [comment]
Let's go with a Hangout

**Comment by mihaiep** [comment]
do we have a working link to join? thanks

**Comment by mhdawson** [comment]
Link for participants: https://hangouts.google.com/hangouts/_/ytl/PbVbxR8xgZuZ7pgV7EFqNea8G7myLagUQkkwDPNDL8w=?eid=100598160817214911030

**Comment by mihaiep** [comment]
Tracking: Dan could you open a new issue for tracking with what can we make public from the Benchmarking WG Survey survey (per our meeting today), and try to invite Tracy for the next meeting  User feedback meeting 1.26.2018?thank you

**Comment by dshaw** [comment]
The minutes from this meeting need to be PRed to the repo:
https://docs.google.com/document/d/1igMLOTd0TJHf0hbXqcLIl-tXmj9vlK9G3h1W2d9TQFo/edit
